### DISCUSSION TITLE
repo is public, site built successfully and gh-pages branch exists and looks good, but still "site not found"

### DISCUSSION THREAD

**Discussion by rosskarchner** [community/community#123771] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I think I've done all of the troubleshooting possible to publish a GitHub site for this repo: https://github.com/rosskarchner/pong-wars-godot

Visibility is public, custom domain is verified and set up correctly, the build is successful and the gh-pages branch contains an index.html. But both rosskarchner.github.io/pong-wars-godot/ and pongwars.karchner.com (the custom domain) return "site not found".

Any ideas for what I should look at next?

**Comment by LiteBrite82** [comment]
@rosskarchner 
[rosskarchner.itch.io/pong-wars](https://rosskarchner.itch.io/pong-wars) works for me.
pongwars.karchner.com gets a 404

**Comment by rosskarchner** [comment]
> @rosskarchner 
> [rosskarchner.itch.io/pong-wars](https://rosskarchner.itch.io/pong-wars) works for me.
> pongwars.karchner.com gets a 404

Yeah-- Itch is an alternate hosting service I'm using while I troubleshoot Github pages

**Reply by LiteBrite82** [reply]
ok... digging deeper. 
on the plus side, the 404 for the custom domain is a GitHub 404; so we are headed in the right direction. in the github settings for that repo, under pages, you have the deployment branch set to gh-pages and the path set to /root ?

**Reply by LiteBrite82** [reply]
I saw that change! now the deployment is pointing at the main branch and showing the readme.md by default.

**Reply by rosskarchner** [reply, marked as answer]
The README thing happened by accident-- but I figured out how to actually make it work the way I wanted. On my pages config, I had it set to deploy via "github actions", which I assumed would just work because I was indeed using a GitHub action to push the built site to my gh-pages branch.

After reading the docs for JamesIves/github-pages-deploy-action@releases/v4 I realized that wasn't enough-- I had to configure pages to deploy the gh-pages branch. All good now!

**Reply by LiteBrite82** [reply]
I cloned your gh-pages branch and was able to [successfully deploy](https://github.com/LiteBrite82/pong-test). though there are errors with loading resources. its likely an issue with file pathing in the prebuild configuration. 

![Screenshot 2024-05-14 at 11 39 39 AM](https://github.com/community/community/assets/163985914/93ae6b3a-9de3-4535-b56d-8dcdc83a1618)

**Reply by LiteBrite82** [reply]
NIce!!!

**Reply by LiteBrite82** [reply]
wait, no, that missing file is my bad - I cloned the quick and dirty way through the browser and missed copying the file.

**Reply by rosskarchner** [reply]
Thanks for your help!
